src/pkcs15init/pkcs15-lib.c

```diff
@@ -1238,7 +1238,13 @@ sc_pkcs15init_init_prkdf(struct sc_pkcs15_card *p15card, struct sc_profile *prof
 		key_info->params.data = &keyargs->key.u.ec.params;
 		key_info->params.free_params = sc_pkcs15init_empty_callback;
 		key_info->field_length = ecparams->field_length;
-		key_info->modulus_length = 0;
```

Comment:
According to ISO 7816-15 (successor of PKCS#15), an `PublicECKeyAttributes` doesn't have a `modulusLength` field (while a `
PublicRSAKeyAttributes` *does* have one):
![image](https://user-images.githubusercontent.com/1266577/28341066-5970e912-6c12-11e7-97da-c2ea782a6180.png)
![image](https://user-images.githubusercontent.com/1266577/28341093-75b83882-6c12-11e7-8537-bfabba75ff90.png)

What's the reason for this change? As far as I can see, you're not using the `modulus_length` somewhere else.